src/components/prediction/DownloadData.tsx

```diff
@@ -0,0 +1,76 @@
+ import PropTypes from "prop-types";
+import { useEffect, useState } from "react";
+import JSZip from "jszip";
+import { saveAs } from "file-saver";
+import useAnalyticRequest from "../../hooks/useAnalyticRequest";
+import useOrgUnits from "../../hooks/useOrgUnits";
+import React from "react";
+
+interface DownloadDataProps {
+  period: any;
+  orgUnitLevels: any[];
+  orgUnits: any;
+  temperatureData: any;
+  precipitationData: any;
+  populationData: any;
+  setStartDownload: (startDownload : boolean) => void
+}
+
+const DownloadData = ({ period, setStartDownload, orgUnitLevels, orgUnits, temperatureData, precipitationData, populationData}: DownloadDataProps) => {
+  
+  //Concat selected orgUnits (either national, a district, chiefdom or facility) with the id of selected levels (districts, chiefdoms, facilities)
+  const mergedOrgUnits = orgUnitLevels.map(level => "LEVEL-"+level).join(";")+";"+orgUnits.map((ou : any) => ou.id).join(";")
+
+  //Convert the period selected to a DHIS2-standard list of months
+  const fillPeriodesMonths = (period : any) => {
```

Comment:
This is fine for now, just remember that we need to support multiple period types in the future.